darshan-util/pydarshan/darshan/tests/test_plot_exp_common.py

```diff
@@ -366,13 +366,13 @@ def test_xticks_and_labels(log_path, func, expected_xticklabels, mod):
             [0, 3, 3, 0, 0, 0],
         ),
         (
-            "shane_ior-PNETCDF_id864223-864223_10-27-46849-11258636277699483231_1.darshan",
+            "shane_ior-PNETCDF_id438100-438100_11-9-41525-10280033558448664385_1.darshan",
             "PNETCDF_FILE",
             plot_opcounts,
             [0, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0],
         ),
         (
-            "shane_ior-PNETCDF_id864223-864223_10-27-46849-11258636277699483231_1.darshan",
+            "shane_ior-PNETCDF_id438100-438100_11-9-41525-10280033558448664385_1.darshan",
```

Comment:
is there much benefit to the turnover of log files here and elsewhere vs. adding to logs repo and flushing through the new logs that way?

Reply:
the only reason i switched log files again is that the old ones were generated by a previous development branch (pnetcdf) that we will no longer be able to parse correctly on this branch. essentially, the old logs have the same embedded format versions as the logs in this PR (we only really bump them on release), but the logs do have slightly different underlying formats. this means logutils code can't properly parse the old versions unfortunately.

this is one of the drawbacks on testing logs that were not generated by stable darshan releases (as opposed to dev branches), as we can't reliably guarantee backwards compatibility between releases (well, i guess we could, but we won't).